src/scenes/level_scene.py

```diff
@@ -738,7 +738,7 @@ def get_next_cases(self, position: Position) -> list[Optional[Entity]]:
         return tiles_content
 
     def get_possible_moves(
-        self, position: Position, max_moves: int
+        self, position: tuple, max_moves: int
```

Comment:
feel like we are loosing some level of precision about the structure of position arg here

Reply:
This change doesn't make sense, must've been an overzealous mistake on my part.